docs/index.rst

```diff
@@ -0,0 +1,48 @@
+casa-formats-io documentation
+=============================
+
+Scope
+-----
+
+The **casa-formats-io** package is a small package which implements
+functionality to read data stored in CASA formats (such as .image datasets).
+This implementation is independent of and does not use `casacore
+<https://casacore.github.io/casacore/>`_. The motivation for this package is
+to provide:
+
+* Efficient data access via `dask <https://dask.org/>`_ arrays
+* Cross-platform data access, supporting Linux, MacOS X and Windows
+* Data access with all modern Python versions, from 3.6 to the latest Python version
+
+At this time, only reading .image datasets is supported. Reading measurement sets
+(.ms) or writing data of any kind are not yet supported.
+
+Using casa-formats-io
+---------------------
+
+To construct a dask array backed by a .image dataset, use the
+:func:`~casa_io_formats.image_to_dask` function::
+
+    >>> from casa_formats_io.casa_dask import image_to_dask
+    >>> image_to_dask('my_dataset.image/')
```

Comment:
I think it would helpful to show setting this to a variable. Otherwise it looks like it's an in place operation like CASA's imagetools or `ia`

```suggestion
    >>> dask_array = image_to_dask('my_dataset.image/')
```

Reply:
Good point thanks!